indico/util/countries.py

```diff
@@ -12,9 +12,11 @@
 
 from indico.core.config import config
 from indico.util.i18n import get_current_locale
+from indico.util.signals import make_interceptable
 from indico.util.string import remove_accents, str_to_ascii
 
 
+@make_interceptable
```

Comment:
What's the usecase for this which isn't covered by the existing `CUSTOM_COUNTRIES` functionality?

Reply:
We have a whole table where we defined countries and their translations so we're doing a db query there, not sure we can put it in the config

Reply:
OK, fine